multiversx/mx-api-service
# Fix account token idnetifier if fetched from ElasticSearch

## Proposed Changes
- Make sure the `identifier` is set correctly when fetching MetaESDTs from ElasticSearch using the `includeMetaESDTs` parameter

## How to test (devnet)
- in redis cli, use the following command: `SET addressEsdtTrieTimeout:erd1xlzeycr9kn9v5ysaj2td347t9ll6uacv0t5q4pvq0yz6e35l44esw7wkx5 "true" EX 3600`
- after a service restart, the `/accounts/erd1xlzeycr9kn9v5ysaj2td347t9ll6uacv0t5q4pvq0yz6e35l44esw7wkx5/tokens?includeMetaESDT=true` route should also include nonce in token identifier
